packages/http/httpcall_common.js

```diff
@@ -5,9 +5,14 @@ makeErrorByStatus = function(statusCode, content) {
     return str.length > length ? str.slice(0, length) + '...' : str;
   };
 
+  // this solution is not browser compatible
+  // var contentToCheck = typeof Buffer != "undefined" && content instanceof Buffer ? content.toString() : content;
+  var contentToCheck = typeof content == "string" ? content : content.toString();
```

Comment:
But this time we forgot other text format like Json or Xml. Maybe we should just not stringified Buffer and just let the failing message.